src/test/java/com/wikia/webdriver/testcases/mercurytests/CategoryPageTest.java

```diff
@@ -45,10 +48,25 @@ public void mercury_category_navigateThroughSection() {
         .loadPreviousMembersForSection("C");
   }
 
-  @Test(groups = "mercury_category_navigateToCategoryMemberPage(")
+  @Test(groups = "mercury_category_navigateToCategoryMemberPage")
   public void mercury_category_navigateToCategoryMemberPage() {
     new CategoryPage()
         .navigateToPageWithArticleAndWithMembersFromUrl()
         .navigateToCategoryMemberPage("Category test 001");
   }
+
+  @Test(
+          groups = "mercury_category_title_present",
+          dataProvider = "getCategoriesWithTitles",
+          dataProviderClass = MercuryCategoriesDataProvider.class
+  )
+  public void mercury_category_title_present(String pageUrl, String expectedTitle) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)